terraform/install.sh

```diff
@@ -204,7 +214,9 @@ installMonitoring() {
 
   gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
   log "Please create a monitoring workspace for the project by clicking on the following link: $gcp_monitoring_path"
```

Comment:
This line should also go into if in line 217.

Reply:
Good catch!